paasta_tools/docker_wrapper.py

```diff
@@ -341,6 +343,10 @@ def main(argv=None):
 
     if mesos_task_id and can_add_hostname(argv):
         hostname = generate_hostname(socket.getfqdn(), mesos_task_id)
+        # Add PAASTA_HOST environment variable
+        if env_file_name is not None:
```

Comment:
I thought the env_file_name was only used in the cases where we have secrets. 

Instead of appending to this file, can you instead "just" add an additional env arg?